sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/AttestationClient.java

```diff
@@ -9,31 +9,78 @@
 import com.azure.core.annotation.ServiceMethod;
 import com.azure.core.http.rest.Response;
 import com.azure.core.util.Context;
-import com.azure.security.attestation.implementation.AttestationsImpl;
-import com.azure.security.attestation.implementation.models.TpmAttestationRequest;
-import com.azure.security.attestation.implementation.models.TpmAttestationResponse;
 import com.azure.security.attestation.models.AttestOpenEnclaveRequest;
 import com.azure.security.attestation.models.AttestSgxEnclaveRequest;
 import com.azure.security.attestation.models.AttestationResponse;
+import com.azure.security.attestation.models.AttestationSigner;
 import com.azure.security.attestation.models.CloudErrorException;
 
-import java.nio.charset.StandardCharsets;
-import java.util.Objects;
+import static com.azure.core.util.FluxUtil.withContext;
 
-/** Initializes a new instance of the synchronous AzureAttestationRestClient type. */
+/** Initializes a new instance of the synchronous AttestationClient object. */
 @ServiceClient(builder = AttestationClientBuilder.class)
 public final class AttestationClient {
-    private final AttestationsImpl serviceClient;
+    private final AttestationAsyncClient asyncClient;
 
     /**
      * Initializes an instance of Attestations client.
      *
      * @param serviceClient the service client implementation.
      */
-    AttestationClient(AttestationsImpl serviceClient) {
-        this.serviceClient = serviceClient;
+    AttestationClient(AttestationAsyncClient serviceClient) {
+        this.asyncClient = serviceClient;
     }
 
+    /**
+     * Retrieves the OpenId Metadata for this AttestationClient instance.
+     * @return Object containing the OpenId metadata configuration for this instance.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Object getOpenIdMetadata() {
```

Comment:
>getOpenIdMetadata [](http://example.com/codeflow?start=18&length=17)

You should change this to  this.getOpenIdMetadataWithResponse(Context.NONE).getValue(); --> we do not bring flux's context to sync APIs.

Reply:
Fixed. Also the other instances in the file.